docs/api_extra.rst

```diff
@@ -698,14 +709,6 @@ section <ndarrays>`.
       ``shape()``, ``stride()``, and ``operator()`` following the conventions
       of the `ndarray` type.
 
-   .. cpp:function:: template <typename... Ts> auto& operator()(Ts... indices)
```

Comment:
What's the reason for this removal? AFAIK this operator is still there, and it should be documented.